lib/date-utils.ts

```
@@ -0,0 +1,44 @@
+ * @param fallbackDate - The fallback date to use if the input is invalid (defaults to '1970-01-01')
+ * @returns A valid ISO date string or the fallback date
+ *
+ * @example
+ * normalizeDate('2024-12-10') // returns '2024-12-10'
+ * normalizeDate('invalid-date') // returns '1970-01-01'
+ * normalizeDate(undefined) // returns '1970-01-01'
+ * normalizeDate('', '2024-01-01') // returns '2024-01-01'
+ */
+export function normalizeDate(
+  dateString?: string,
+  fallbackDate: string = '1970-01-01'
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)